restcomm/restcomm.sms/src/main/java/org/restcomm/connect/sms/SmsSession.java

```diff
@@ -235,6 +236,13 @@ public void onReceive(final Object message) throws Exception {
             final SmsSessionAttribute attribute = (SmsSessionAttribute) message;
             attributes.put(attribute.name(), attribute.value());
             Object record = attributes.get("record");
+            try {
```

Comment:
is this try necesasary to test the casting, im not sure i like the construct i would prefer to use instanceof